packages/vue/src/components/molecules/SfAddToCart/SfAddToCart.js

```diff
@@ -0,0 +1,37 @@
+import SfButton from "../../atoms/SfButton/SfButton.vue";
+import SfInput from "../../atoms/SfInput/SfInput.vue";
+
+export default {
+  name: "SfAddToCart",
+  components: {
+    SfButton,
+    SfInput
+  },
+  model: {
+    prop: "qty"
+  },
+  props: {
+    /**
+     * Boolean to indicate whether product
+     * can be added to cart
+     */
+    canAddToCart: {
+      type: Boolean,
+      default: false
+    },
+    /**
+     * Stock quantity of product
+     */
+    quantity: {
```

Comment:
```suggestion
    stock: {
```
maybe we should name it stock, stockQuantity or something like that

Reply:
stock is good :)